firebase-storage/test-app/src/main/res/layout/content_main.xml

```diff
@@ -286,13 +286,24 @@
 
     <!-- Tenth Row -->
 
+    <Button
+        android:textSize="7sp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Use Emulator"
+        android:id="@+id/useEmulator"
+        android:layout_below="@+id/listSinglePage"
+        android:layout_alignParentLeft="true" />
+
+    <!-- Eleventh Row -->
```

Comment:
Oh, you uncovered the darkest secret for the Storage SDK - the button layout in the test app. I hope you enjoyed it? :)

Reply:
I had to zoom my screen a lot to see those tiny buttons.